src/SignalR/clients/ts/signalr/src/empty.ts

```diff
@@ -0,0 +1,13 @@
+import { HttpClient, HttpResponse } from "./HttpClient";
+import { ILogger } from "./ILogger";
+
+export class NodeHttpClient extends HttpClient {
+    // @ts-ignore
+    public constructor(logger: ILogger) {
+        super();
+    }
+
+    public send(): Promise<HttpResponse> {
+        throw new Error("Method not implemented.");
```

Comment:
This error message could be better too. It should probably recommend the correct way to consume the package from node.js.